packages/server/api/src/app/ai/chat/ai-mcp-chat.controller.ts

```diff
@@ -379,7 +381,7 @@ async function getUserMessage(
       return null;
     }
 
-    return String(firstContentElement.text);
+    return String(firstContentElement.text ?? 'continue');
```

Comment:
If it's a frontend tool, we just say `continue` to the conversation (this will be shown as a user message if the user reloads).